cli/remove_a_pool.md

```diff
@@ -0,0 +1,15 @@
+# Removing a pool
+To removing a pool is as simple as removing a prefix.
+
+All prefixes are dismembered from the pool, they do not belong to a pool
```

Comment:
I think this should be reworded. "dismembered" means something different ;)

"You can remove a pool with the following command. Current member prefixes of the pool will be disassociated from the pool but continue to exist."